core/src/main/java/io/questdb/std/MemoryTag.java

```diff
@@ -35,15 +35,18 @@
     public static final int MMAP_TABLE_READER = 7;
     public static final int NATIVE_COMPACT_MAP = 8;
     public static final int NATIVE_FAST_MAP = 9;
-    public static final int NATIVE_LONG_LIST = 10;
+    public static final int NATIVE_FAST_MAP_LONG_LIST = 10;
     public static final int NATIVE_HTTP_CONN = 11;
     public static final int NATIVE_PGW_CONN = 12;
     public static final int MMAP_INDEX_READER = 13;
     public static final int MMAP_INDEX_WRITER = 14;
     public static final int MMAP_INDEX_SLIDER = 15;
     public static final int MMAP_BLOCK_WRITER = 16;
     public static final int NATIVE_REPL = 17;
-    public static final int SIZE = NATIVE_REPL + 1;
+    public static final int NATIVE_SAMPLE_BY_LONG_LIST = 18;
+    public static final int NATIVE_LATEST_BY_LONG_LIST = 19;
+    public static final int NATIVE_LONG_LIST = 20;
```

Comment:
nit: do we really need this tag considering that it's only used in tests?

Reply:
Yeah, this is an odd-ball. I removed it initially, then stumbled on standalone usage in tests. Honestly I didn’t want to use random tag in tests; that felt odd. Perhaps there will be standalone usage in prod code in the future

Reply:
OK, let's keep it then. My only concern is that it's tempting to start using it somewhere in the code base instead of a specialized tag.